specifications/ticket-api.yaml

```diff
@@ -0,0 +1,401 @@
+openapi: 3.0.0
+info:
+  description:
+    API Example for ticket sales. The API provides methods for checking the
+    availability and pricing of tickets, to identify the user and to deliver the
+    digital ticket in various formats. The API does not handle routing and
+    relies on unique itinerary id provided by external routing API before 
+    calling the ticket query.<br/>
+    <br/>
+    Copyright 2017 FLOU Ltd.
+  version: "0.1.0-oas3"
+  title: Ticket API
+  contact:
+    name: FLOU
+    url: 'https://www.flou.io/'
+    email: sami.makinen@flou.io
+  license:
+    name: Apache License 2.0
+    url: 'https://github.com/maas-alliance/apis/LICENSE'
+tags:
+  - name: purchase
+    description: Handle ticket queries and purchases
+paths:
+  /tickets/query:
+    post:
+      tags:
+        - purchase
+      summary: Query available transactions
+      description: List available ticket options for the specified itinerary
+      operationId: queryTransactions
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/transactions'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/transactions'
+        '400':
+          description: Invalid query parameter
+        '403':
+          description: Query forbidden
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/query-parameters'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/query-parameters'
+  /tickets/confirm:
+    post:
+      tags:
+        - purchase
+      summary: Confirm transaction
+      description: Confirm provided transaction and receive a ticket product
+      operationId: condirmTransaction
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ticket-response'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ticket-response'
+        '400':
+          description: Invalid parameters
+        '403':
+          description: Transaction denied
+        '404':
+          description: Service not found
+      requestBody:
+        content:
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/confirm-parameters'
+          application/json:
+            schema:
+              $ref: '#/components/schemas/confirm-parameters'
+
+components:
+  schemas:
+    geo-point:
```

Comment:
Could also unify this with https://github.com/maas-alliance/apis/blob/7b32d9e966e351f694738fb4ccf3d9c6eac20bc9/specifications/booking.yaml#L494. Would be nice to have a separate `common-schemas.yaml` or so. Anyone know if this is possible? @MarianoTucat?

Reply:
I would like to figure out the `common-schemas.yaml` approach.

Reply:
Lets move this into a separate issue.